internal/handler/wallet/create.go

```diff
@@ -42,20 +42,17 @@ func HandleCreate(params operations.RestWalletCreateParams) middleware.Responder
 
 func New(newWallet *wallet.Wallet) middleware.Responder {
 
-	privK := base58.CheckEncode(newWallet.KeyPair.PrivateKey, wallet.Base58Version)
-	pubK := base58.CheckEncode(newWallet.KeyPair.PublicKey, wallet.Base58Version)
-	salt := base58.CheckEncode(newWallet.KeyPair.Salt[:], wallet.Base58Version)
-	nonce := base58.CheckEncode(newWallet.KeyPair.Nonce[:], wallet.Base58Version)
+	pubK := "P" + base58.CheckEncode(newWallet.KeyPair.PublicKey, wallet.Base58Version)
```

Comment:
this should be factorized because it appears several times it will also allow to set an outstanding name for the function. and "P" is a magic string that should be centralized